packages/zowe-explorer/__tests__/__unit__/trees/dataset/DatasetFSProvider.unit.test.ts

```diff
@@ -215,6 +216,15 @@ describe("fetchDatasetAtUri", () => {
                 };
             }),
         };
+        Object.defineProperty(Profiles, "getInstance", {
+            value: jest.fn(() => {
+                return {
+                    loadNamedProfile: jest.fn(() => {
+                        return testProfile;
+                    }),
+                };
+            }),
+        });
```

Comment:
would we be able to cut down on the duplication added to this file?  the 3 following additions are identical to this one, can it go at a global or block level to be used?